qiskit_experiments/framework/composite/parallel_experiment.py

```diff
@@ -136,6 +163,18 @@ def _combined_circuits(self, device_layout: bool) -> List[QuantumCircuit]:
                 # Apply transpiled subcircuit
                 # Note that this assumes the circuit was not expanded to use
                 # any qubits outside the specified physical qubits
+                circ_duration = max_durations.get(circ_idx)
+                pad_time = None
+                if scheduling_method and sub_circ.duration and circ_duration:
+                    pad_time = abs(circ_duration - sub_circ.duration)
```

Comment:
Same here.

Reply:
the if check for `sub_circ.duration and circ_duration` prevents this